control/summary_table_test.go

```diff
@@ -81,16 +80,4 @@ var _ = Describe("SummaryTable", func() {
 			Expect(origination.origins[sharedOrigination].IsChecked()).To(Equal(true))
 		})
 	})
-
-	Describe("Diff", func() {
-		It("detects no diff when the value of responsible role is empty", func() {
-			table := getTable("AC-2")
-			st := NewSummaryTable(table)
-			openControlData := fixtures.LoadOpenControlFixture()
-			diff, err := st.Diff(openControlData)
-
-			Expect(diff).To(Equal([]reporter.Reporter{}))
-			Expect(err).To(BeNil())
-		})
-	})
```

Comment:
it's useful but the diff won't be empty anymore AC-2 because it'll have a diff with control origin. I'll put it back but put a skip and explain why.